flowdb/sql/04_schema_other.sql

```diff
@@ -80,3 +80,22 @@ CREATE TABLE etl.etl (
 	time_stamp TIMESTAMP WITH TIME ZONE,
 	PRIMARY KEY (id)
 );
+
+/*
+
+Schema used for record keeping of aggregate calculations.
+
+*/
+
+CREATE SCHEMA IF NOT EXISTS aggregates;
+
+CREATE TYPE aggstatus AS ENUM ('in_process', 'done', 'failed');
```

Comment:
Not needed for this PR, but you may want to rename the existing `status` type to `ingestion_status` in order to distinguish it from `aggstatus`.

Reply:
Yep - I should